src/db/models/ArticleTag.js

```diff
@@ -0,0 +1,52 @@
+export default (Sequelize, DataTypes) => {
+  const Tags = Sequelize.define(
+    'ArticleTags',
+    {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+      },
+      tagId: {
+        allowNull: false,
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
```

Comment:
I don't think this column requires a default value since it is a foreign key on the ```Tags``` table

Reply:
Yeah, you are right